docs/multi_process.md

```diff
@@ -28,12 +28,15 @@ Key concepts:
   * Each process has a
     distinct set of *local* devices it can address. The *global* devices are the set
     of all devices across all processes.
-  * Use standard JAX parallelism APIs like {func}`~jax.pmap` and
-    {func}`~jax.experimental.maps.xmap`. Each process “sees” *local* input and
-    output to parallelized functions, but communication inside the computations
-    is *global*.
+  * Use standard JAX parallelism APIs like {func}`~jax.jit` (see
+    {doc}`/sharded-computation` tutorial) and
+    {func}`~jax.experimental.shard_map.shard_map`. Each process “sees”
+    *local* input and output to parallelized functions, but
```

Comment:
I don't think the local thing is true for `jit`. Or rather the way this sentence is phrased is confusing.

Reply:
It is true, but it's confusing. You have local data but global shapes.

Reply:
I don't think we need to talk about local data. I think it's only shapes that matter. For jit, it's always global shapes and for shard_map it's *device* local shape.

Reply:
This sentence was already there. I'm not sure how to re-write it.
Any suggestion? Or I can remove it or let that for another PR.

Otherwise, I fixed all comments.